shell/platform/android/io/flutter/embedding/android/FlutterActivity.java

```diff
@@ -450,6 +450,23 @@ public FlutterActivity() {
     this.delegate = delegate;
   }
 
+  /**
+   * Returns the {@link ExclusiveAppComponent<Activity>} that is associated with {@link
+   * io.flutter.embedding.engine.FlutterEngine}.
+   *
+   * <p>In the scenario where multiple {@link FlutterActivity} share the same {@link FlutterEngine},
+   * to attach/re-attache a {@link FlutterActivity} to the shared {@link FlutterEngine}, we MUST
+   * manually invoke {@link ActivityControlSurface#attachToActivity(ExclusiveAppComponent,
+   * Lifecycle)}.
+   *
+   * <p>The {@link ExclusiveAppComponent} is exposed here so that subclasses of {@link
```

Comment:
nit: when this javadoc is specified in the interface, there's no need to copy it again in each implementation of the interface

Reply:
thx, I got it.